pkg/limayaml/limayaml.go

```diff
@@ -215,6 +218,7 @@ type Provision struct {
 	Mode                            ProvisionMode `yaml:"mode,omitempty" json:"mode,omitempty" jsonschema:"default=system"`
 	SkipDefaultDependencyResolution *bool         `yaml:"skipDefaultDependencyResolution,omitempty" json:"skipDefaultDependencyResolution,omitempty"`
 	Script                          string        `yaml:"script" json:"script"`
+	File                            *string       `yaml:"file,omitempty" json:"file,omitempty" jsonschema:"nullable"`
```

Comment:
Same, needs to support digest

Reply:
Again, this can be added later when we need it.

Reply:
Yep, but at least we have to decide whether we will just use the existing file locator structure, or we will have to introduce a new microformatted string
- https://github.com/lima-vm/lima/discussions/3019

Reply:
Let's have the discussion over there for now. It is not urgent to get this PR merged; we can take our time to discuss it.

I just want to keep the discussion is a separate "discussion", so it doesn't get buried when this PR gets closed.